src/App.tsx

```diff
@@ -1,24 +1,244 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import { nanoid } from 'nanoid';
+import React, {
+  useEffect, useState, useMemo, useRef,
+} from 'react';
+import { Todo, SortType } from './Types';
+import { UserWarning } from './components/UserWarning';
+import { getTodos } from './todos';
+import { client } from './utils/client';
+import { ErrorMessages } from './components/ErrorMessages';
```

Comment:
Rename the file `errorMessages.tsx` to `ErrorMessages.tsx`
![image](https://github.com/mate-academy/react_todo-app-with-api/assets/99403198/0704a93e-e436-443d-8d8a-3a8bf72dc128)